src/tools/pontoon/ui.md

```diff
@@ -111,7 +117,7 @@ Built-in translation tools are located in the rightmost column. For more informa
 
 #### Terminogy
 
-The `TERMS` tab shows the definition for a term, in case the source string includes matches with the built-in terminology. The matching term is also highlighted in the source string.
+The `TERMS` tab shows the definition for a term, in case the source string includes matches with the built-in [terminology](glossary.md#terminology). The matching term is also highlighted in the source string.
```

Comment:
Note: I'm using links to the glossary page from #213, which might be broken if we merge this first.